yml/cdb_startrackerfast.yaml

```diff
@@ -0,0 +1,381 @@
+    description: Dome azimuth.
+  - name: vignette
+    "@id": "#exposure.vignette"
+    datatype: string
+    length: 10
+    description: "Instrument blocked from the sky: UNKNOWN, NO, PARTIALLY, FULLY."
+  - name: vignette_min
+    "@id": "#exposure.vignette_min"
+    datatype: string
+    length: 10
+    description: "Lowest amount of instrument vignetting detected during the exposure: UNKNOWN, NO, PARTIALLY, FULLY."
+- name: exposure_flexdata
+  "@id": "#exposure_flexdata"
+  primaryKey:
+  - "#exposure_flexdata.obs_id"
+  - "#exposure_flexdata.key"
+  constraints:
+  - name: fk_exposure_flexdata_obs_id
+    "@id": "#exposure_flexdata.fk_obs_id"
+    "@type": ForeignKey
+    description: Flex data obs_id must be an Exposure exposure_id.
+    columns: ["#exposure_flexdata.obs_id"]
+    referencedColumns: ["#exposure.exposure_id"]
+  - name: fk_exposure_flexdata_key
+    "@id": "#exposure_flexdata.fk_key"
+    "@type": ForeignKey
+    description: Flex data key must be listed in the schema.
+    columns: ["#exposure_flexdata.key"]
+    referencedColumns: ["#exposure_flexdata_schema.key"]
+  columns:
+  - name: obs_id
+    "@id": "#exposure_flexdata.obs_id"
+    datatype: long
+    nullable: false
+    description: Unique identifier.
+  - name: key
+    "@id": "#exposure_flexdata.key"
+    datatype: string
+    length: 128
+    nullable: false
+    description: Name of key.
+  - name: value
+    "@id": "#exposure_flexdata.value"
+    datatype: text
+    description: Content of value as a string.
+- name: exposure_flexdata_schema
+  "@id": "#exposure_flexdata_schema"
+  primaryKey:
+  - "#exposure_flexdata_schema.key"
+  columns:
+  - name: key
+    "@id": "#exposure_flexdata_schema.key"
+    datatype: string
+    length: 128
+    nullable: false
+    description: Name of key.
+  - name: dtype
+    "@id": "#exposure_flexdata_schema.dtype"
+    datatype: string
+    length: 64
+    nullable: false
+    description: Name of the data type of the value, one of bool, int, float, str.
+  - name: doc
+    "@id": "#exposure_flexdata_schema.doc"
+    datatype: text
+    nullable: false
+    description: Documentation string.
+  - name: unit
+    "@id": "#exposure_flexdata_schema.unit"
+    datatype: string
+    length: 128
+    description: Unit for the value. Should be from the IVOA (https://www.ivoa.net/documents/VOUnits/) or astropy.
+  - name: ucd
+    "@id": "#exposure_flexdata_schema.ucd"
+    datatype: string
+    length: 128
+    description: IVOA Unified Content Descriptor (https://www.ivoa.net/documents/UCD1+/).
+- name: exposure_quicklook
+  "@id": "#exposure_quicklook"
+  primaryKey:
+  - "#exposure_quicklook.visit_id"
+  constraints:
+  - name: fk_exposure_quicklook_obs_id
+    "@id": "#exposure_quicklook.fk_obs_id"
+    "@type": ForeignKey
+    description: Quicklook visit_id must be an Exposure exposure_id.
+    columns: ["#exposure_quicklook.visit_id"]
+    referencedColumns: ["#exposure.exposure_id"]
+  columns:
+  - name: visit_id
```

Comment:
`exposure` tables shouldn't have any references to `visit`, especially since the Star Trackers don't do snaps.  This should be `exposure_id`.